spec/attribute.dd

```diff
@@ -722,7 +685,13 @@ $(GNAME UserDefinedAttribute):
     $(D @) $(GLINK2 template, TemplateInstance) $(D $(LPAREN)) $(GLINK2 expression, ArgumentList)$(OPT) $(D $(RPAREN))
 )
 
-And looks like:
+    $(P
+        User Defined Attributes (UDA) are compile-time expressions that can be attached
+        to a declaration. These attributes can then be queried, extracted, and manipulated
+        at compile time. There is no runtime component to them.
+    )
+
+A user-defined attribute looks like:
```

Comment:
@andralex [suggested](https://github.com/dlang/dlang.org/pull/1040/files#r39329490) to use "user-defined attributes" .
What's the official spelling: with dash or without?

Reply:
dash please